src/api/app/views/webui/repositories/create.js.erb

```diff
@@ -1 +1 @@
-$('#flash-messages').replaceWith('<%= escape_javascript( render layout: false, partial: 'layouts/webui/flash', object: flash, formats: [ :html ]) %>');
\ No newline at end of file
+$('#flash').html("<%= escape_javascript(render(layout: false, partial: 'layouts/webui2/flash', object: flash)) %>");
```

Comment:
```
$('#flash').html('#{escape_javascript(render(...)}')
```
would allow you to migrate this view to  haml

Reply:
I don't have a strong opinion on this, but sure, I can do it.